packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -235,24 +241,30 @@ hub.post("/", async (req, res) => {
           try {
             return {
               spokeName: spokeName,
-              errorReported: errorOutput.errorOutputs[spokeName].execResponse
-                ? errorOutput.errorOutputs[spokeName].execResponse.stderr
-                : errorOutput.errorOutputs[spokeName],
+              errorReported:
```

Comment:
this is the one main change made here to try and more consistently produce error logs. in different testing situations the previous block would return blank.